extensions/amp-bind/amp-bind.md

```diff
@@ -261,6 +261,8 @@ An `amp-state` element must contain a child `<script>` element.
 
 #### Attributes
 
+[filter formats="websites, stories, ads"]
```

Comment:
It looks like this `filter` line appears both above and below the "Attributes" header with the change, the one above being unclosed.

Reply:
You're right - (our parser doesn't care though). Went ahead and fixed the filters.